src/Routes/patient.routes.js

```diff
@@ -31,13 +31,13 @@ const Bill = require("../models/billModel")
  *                 patientAddress:
  *                     type: object
  *                     properties:
- *                         address_line:
+ *                         addressLine:
  *                             type: string 
- *                         City:
+ *                         city:
  *                             type: string
- *                         Postal_Code:
+ *                         postalCode:
  *                             type: number
- *                         Country:
+ *                         country:
  *                             type: string
```

Comment:
changes all the variable names according to camelCase naming convention in every file